content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -56,12 +56,22 @@ endpoint** in your **Hosted endpoints**. A form opens, where you'll need to add
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+.. _stripe/pmt:
+
 Enable local payment methods
 ----------------------------
 
 Local payment methods are payment methods that are only available for certain merchants and
 customers countries and currencies.
 
+Odoo 15 supports the following local payment methods:
+
+- Bancontact
+- EPS
+- giropay
+- iDEAL
+- Przelewy24 (P24)
```

Comment:
Maybe add some links? It's just a suggestion

```suggestion
- `Bancontact <https://www.bancontact.com>`_
- EPS
- giropay
- iDEAL
- Przelewy24 (P24)
```

Reply:
I don't think that adding some links to the payment methods website is a good idea. Stripe may not implement all functionality of those payment methods and it could be misleading to our users. Anyway, we could add a link to Stripe's payment method documentation but I find it too technical (See [bancontact](https://stripe.com/docs/payments/bancontact)). What do you think?

Reply:
okay no prob then. :)